core/ts/__tests__/Poll.test.ts

```diff
@@ -258,25 +258,25 @@ describe("Poll", function test() {
       BigInt(Math.floor(Date.now() / 1000)),
     );
 
-    it("should throw if this is the first batch and currentMessageBatchIndex is defined", () => {
-      const command = new PCommand(BigInt(user1StateIndex), user1Keypair.pubKey, 0n, 1n, 0n, BigInt(pollId));
+    // it("should throw if this is the first batch and currentMessageBatchIndex is defined", () => {
```

Comment:
should this test be removed or is it a WIP?

Reply:
Not applicable any more.